.github/dependabot.yml

```diff
@@ -1,7 +1,19 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: monthly
-  open-pull-requests-limit: 10
+  # Maintain dependencies for Python
+  - package-ecosystem: pip
+    directory: "/"
+    schedule:
+      interval: monthly
+    assignees:
+      - "ezio-melotti"
+    open-pull-requests-limit: 10
+
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
```

Comment:
Let's put this github-actions one alphabetically before pip, like https://github.com/python/cpython/blob/main/.github/dependabot.yml, makes them easier to compare.

Reply:
Done!